openslides/users/views.py

```diff
@@ -183,8 +184,26 @@ def update(self, request, *args, **kwargs):
         old_permissions = list(group.permissions.all())  # Force evaluation so the perms doesn't change.
 
         response = super().update(request, *args, **kwargs)
+
+        def diff(first, second):
+            """
+            This helper function calculates the difference of two lists: first-second.
+            """
+            second = set(second)
+            return [item for item in first if item not in second]
+
         if response.status_code == 200:
-            inform_changed_permissions(group, given_permissions, old_permissions)
+            new_permissions = diff(given_permissions, old_permissions)
+
+            # some permissions are added
+            if(len(new_permissions) > 0):
+                signal_results = permission_change.send(None, action='added', permissions=new_permissions)
+                collections_to_send = []
+                for (receiver, signal_collections) in signal_results:
+                    if signal_collections:
+                        collections_to_send.extend(signal_collections)
+                #send_collections_to_users(collections_to_send, [user.id for user in group.user_set.all()])
+                # inform_changed_data(what?)
```

Comment:
Ok, if you want to send the objects only to some users then you can't use inform_changed_data. I thought it would be easier just to send some objects to all users.